fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```diff
@@ -404,7 +403,7 @@ public boolean checkPathExist(String path, Map<String, String> properties) {
         }
     }
     
-    public TBrokerFD openReader(String clientId, String path, long startOffset, Map<String, String> properties) {
+    public TBrokerOpenReaderResponse openReader(String clientId, String path, long startOffset, Map<String, String> properties) {
```

Comment:
this is a bad modification.

In this class, we can't return a RPC's response. We should handle RPC in RPC layer.

Reply:
I hope get file size when open file success.